github_pr.py

```diff
@@ -87,12 +90,16 @@ def github_list_prs(**args):
     _check_required_fields(['token', 'repo'], **args)
     gh = Github(args['token'])
     repo = gh.get_repo(args['repo'])
+    list_return_obj = 'Null'
```

Comment:
I don't know if I like returning the string 'Null' - looks too much like actual `None`, could be confusing, might make people expect that this fn returns the value `None`. What if we return actual `None` if that's what we want?